server/src/main/java/org/elasticsearch/plugins/PluginsService.java

```diff
@@ -219,10 +214,11 @@ protected List<LoadedPlugin> plugins() {
         return this.plugins;
     }
 
-    private Map<String, LoadedPlugin> loadPluginBundles(PluginsLoader pluginsLoader) {
+    private Map<String, LoadedPlugin> loadPluginBundles(Settings settings, Path configPath, PluginsLoader pluginsLoader) {
         Map<String, LoadedPlugin> loadedPlugins = new LinkedHashMap<>();
-        for (LoadedPluginLayer loadedPluginLayer : pluginsLoader.loadPluginLayers().values()) {
-            loadBundle(loadedPluginLayer, loadedPlugins, settings, configPath);
+
+        for (PluginLayer pluginLayer : pluginsLoader.pluginLayers().toList()) {
```

Comment:
Super nit: it could be pluginsLoader.pluginLayers().forEach() to avoid the copy, but should be OK anyway (it's a small collection of at most 10s of elements)

Reply:
:facepalm: YES! Changed.